hurumap/tests.py

```diff
@@ -0,0 +1,26 @@
+import requests
+
+from django.test import TestCase
+
+
+class MapitIntegration(TestCase):
+    def test_get_kenya_country_geo(self):
+        resp = requests.get('https://mapit.hurumap.org/areas/COUNTRY?generation=1&country=KE')
```

Comment:
`https://mapit.hurumap.org` don't you want this to run on your current code base by using `django.test.Client` and not the deployed code.

Reply:
@karimkawambwa mapit is not in this codebase, it's an external service